test/data/keepalived.conf.tmpl

```diff
@@ -16,6 +16,12 @@ vrrp_script chk_ingress {
     weight 50
 }
 
+{{if .Cluster.APIProvisioningVIP}}
+vrrp_script chk_api_prov {
+    script "/usr/bin/curl -o /dev/null -kLs https://{{ .Cluster.APIProvisioningVIP }}:2379"
```

Comment:
Shouldn't the script check the local service (e.g: 127.0.0.1:2379)?

Reply:
Yes. I'll change that.